### DISCUSSION TITLE
Issue: Compact Style Controls are too ugly

### DISCUSSION THREAD

**Discussion by hassaanmasood** [microsoft/microsoft-ui-xaml#7431] [General]
The Compact Style controls are too ugly and too small to be used in apps. Applying padding manually does not work for all types of controls. Normal Style controls are too large and reducing padding on them does not make any difference.

The compact style controls should be modified to add small padding to look them nicer and more aligned with the modern UI.

I think controls are too small in compact style, they should somewhere in between the full size controls and the compact style.

**Comment by StephenLPeters** [comment]
The compact styles are still in preview and are not actively being worked on at the moment, thank you for the feedback though.
